base/pkg/entry.jl

```diff
@@ -636,8 +635,8 @@ function build!(pkgs::Vector, errs::Dict, seen::Set=Set())
     mktemp() do errfile, f
         build!(pkgs, seen, errfile)
         while !eof(f)
-            pkg = deserialize(f)
-            err = deserialize(f)
+            pkg = chop(readuntil(f, '\0'))
+            err = chop(readuntil(f, '\0'))
```

Comment:
why `chop`?

Reply:
The result of `readuntil` includes the delimiter. I guess we could change that, since we already made that change to `readline`.